package.json

```
@@ -11,10 +11,11 @@
     "readme:toc": "doctoc README.md --maxlevel 3 --title '## Table of Contents'",
     "test": "jest --colors",
     "pretty-quick": "pretty-quick --staged",
-    "prepublishOnly": "rm -rf dist; babel src --out-dir dist --ignore 'src/__tests__/*'",
+    "prepublishOnly": "rm -rf dist && tsc -p tsconfig.prod.json",
     "semantic-release": "semantic-release",
     "test:coverage": "jest --coverage --colors",
-    "test:watch": "jest --watch --coverage"
+    "test:watch": "jest --watch --coverage",
+    "typescript": "tsc"
```

If this script is for typechecking let's add `noEmit` options
```suggestion
    "typescript": "tsc -noEmit"
```